custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -0,0 +1,294 @@
+"""Support for TaHoma cover - shutters etc."""
```

Comment:
```suggestion
"""Base class for TaHoma covers, shutters, awnings, etc."""
```

Reply:
Why is this part not left in cover.py? as the base class, it would make sense to keep it there.

Reply:
I will